assets/html/scss/documenter/components/_docstring.scss

```
@@ -21,6 +22,10 @@
       background-color: transparent;
     }
 
+    .docstring-article-toggle-button {
+      padding: 0.2rem 0.2rem 0.2rem 0;
```

The rest of the docstring header moves right now when you expand/collapse it (because the icon character is of different width).

```suggestion
      padding: 0.2rem 0.2rem 0.2rem 0;
      min-width: 1.1rem; // to make sure it does not resize when icon changes
```